frontend/src/components/Trigger.tsx

```diff
@@ -14,28 +14,33 @@
  * limitations under the License.
  */
 
-import * as React from 'react';
 import Button from '@material-ui/core/Button';
 import Checkbox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
-import Input from '../atoms/Input';
 import MenuItem from '@material-ui/core/MenuItem';
+import * as React from 'react';
+import { stylesheet } from 'typestyle';
+import { ApiTrigger } from '../apis/job';
+import { HelpButton } from '../atoms/HelpButton';
+import Input from '../atoms/Input';
 import Separator from '../atoms/Separator';
 import { commonCss } from '../Css';
-import { dateToPickerFormat } from '../lib/TriggerUtils';
 import {
-  PeriodicInterval,
-  TriggerType,
-  triggers,
   buildCron,
-  pickersToDate,
   buildTrigger,
+  dateToPickerFormat,
+  PeriodicInterval,
+  pickersToDate,
+  triggers,
+  TriggerType,
 } from '../lib/TriggerUtils';
-import { ApiTrigger } from '../apis/job';
-import { stylesheet } from 'typestyle';
 
 interface TriggerProps {
-  onChange?: (trigger?: ApiTrigger, maxConcurrentRuns?: string) => void;
+  onChange?: (config: {
+    trigger?: ApiTrigger;
+    maxConcurrentRuns?: string;
+    catchup: boolean;
```

Comment:
It seems that in BE, we use nocatchup and in FE we use catchup. They are having opposite values. Should we just use the same in BE and FE to avoid accidental mistake, e.g., like forgetting to take negation when transferring value...

Reply:
A nit comment above. Otherwise /lgtm

Reply:
That's a good point.
The only reason backend uses no catchup is to be backward compatible, so we need false as the default value.

But for UI, I think it's better we use catchup to avoid unnecessary concept coersion, what do you think?

Reply:
Sounds reasonable.